sshuttle/client.py

```diff
@@ -668,7 +694,12 @@ def tcp_connection_is_allowed_conditional(dstip, dstport, srcip, check_acl, chec
         if ipInNetwork(srcip, MS1_CIDR) or ipInNetwork(srcip, MS2_CIDR):
             debug3("TCP source %r is from the microservices subnet\n" % srcip)
             return True
-        
+
+        # allow connection to trusted (onprem) DCs
+        if (dstip == TRUSTED_DC1) or (dstip == TRUSTED_DC2):
```

Comment:
I don't think you want to just return `True` here, you still might want to perform the check for `matches_acl` below. So instead you could just set `check_allowed_sources = False`. Although I suppose you could skip all ACLs as well, something to discuss with the team

Reply:
Already merged this. Do you want this change in now or later is fine? if later then I will create a ticket